modin/experimental/engines/pyarrow_on_ray/frame/partition.py

```diff
@@ -19,11 +21,33 @@
 
 
 class PyarrowOnRayFramePartition(PandasOnRayFramePartition):
+    """
+    Class provides partition interface specific for PyArrow backend and Ray engine.
+
+    Inherits functionality from the ``PandasOnRayFramePartition`` class.
+
+    Parameters
+    ----------
+    object_id : ray.ObjectRef
+        A reference to ``pandas.DataFrame`` that needs to be wrapped with this class.
```

Comment:
@amyskov , a reference to ``pyarrow.Table``. This needs to be fixed as well in multiple places.

Reply:
Ok, issue was created.